pom.xml

```diff
@@ -154,6 +155,17 @@
     </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <!-- Force using the proper version of thrift from the transitive dependencies -->
```

Comment:
Doesn't this force transitive dependencies to use the same version of thrift? I wouldn't say "from the transitive dependencies" since those are the ones that are changed.

Reply:
Actually, this is a solution for the problem of `com.twitter.elephantbird` bringing in an old version of thrift. So, I move this to the submodules where `elephantbird` is used and modify the comment accordingly.